### PR TITLE
enhancement/#115-Creating-UI-to-disconnect/connect-apps

### PR DESCRIPTION
**Pull request by baja156**
GitHub Issue (If applicable): #153 

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the current behavior?
image

The current UI is not polished and not integrated with the normal feedr frontend

## What is the new behavior?
Feed incorporated connect Modal
image

Currently you can't disconnect apps, the button only says so.
The disconnect functionality is also not implemented :)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current upstream repo
- [ ] Added new tests to the test suite for the feature
- [x] Tested the full test suite, and ensured that the feature does not break current build.
- [ ] Docs have been added/updated which fit (for bug fixes / features)
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal)
- [ ] Checked licensing of Frameworks/Libraries (if applicable) 

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by FocalChord** [comment]
I have a few issues at the moment when checking this PR out..

I clicked log in with twitter and gave it my details but when I log in the pop up still remains and the icon of twitter does not change on the main page. See the screenshot below. 

Screen Shot 2020-04-29 at 10 29 24 AM

Another issue is that Reddit shows disconnect for me even tho I didn't give Reddit access to my account..

Another one is that when I log into GIthub the first time, it redirects me back to the main page and the logo is not ticked green indicating it is logged in. Instead I have to refresh the page for this to work. After it is green and I click disconnect, it doesn't ever change back to connect.

File: client/src/components/oauth/RedditConnect.jsx [hunk 1]
```diff
@@ -33,48 +61,55 @@ export const RedditConnect = props => {
   };
 
   // TODO: Whenver using the access token, first check the expires_at key of the cookie and refresh if needed
-  const refreshReddit = () => {
-    const cookie = CookieManager.getUserToken(reddit.name);
-    const refreshToken = JSON.parse(cookie).refresh_token;
+  // const refreshReddit = () => {
```
Comment (Dinith1) [inline client/src/components/oauth/RedditConnect.jsx:64]: Are we planning to refresh the Reddit token?

Comment (baja156) [reply client/src/components/oauth/RedditConnect.jsx:64]: yes we are working on it

Comment (baja156) [reply client/src/components/oauth/RedditConnect.jsx:64]: So the token actually isn't getting used for anything, so refreshing it didn't make sense to me. and i have deleted it

**Comment by FocalChord** [comment]
Looks good now, just one question

When I have connected to all the social networks, GitHub is green whereas the others are blue?Is this intended?

Screen Shot 2020-05-01 at 7 02 00 AM

**Pull request merged**